src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1617,6 +1631,18 @@ public virtual void ClearErrorsAndWarnings()
             OnNodeMessagesClearing();
         }
 
+        /// <summary>
+        /// Clears the info messages that are generated when running the graph,
+        /// the State will be set to ElementState.Dead.
+        /// </summary>
+        public virtual void ClearInfoMessages()
+        {
+            infos.RemoveWhere(x => x.State == ElementState.Info);
+            State = ElementState.Dead;
+            ClearTooltipText();
+            OnNodeInfoMessagesClearing();
```

Comment:
I have created a new method OnNodeInfoMessagesClearing to clear the info messages, instead of using ClearErrorsAndWarnings. The reason being, when we want to clear the info message for a node in future, it should not clear any current errors/warnings for that node.

Reply:
I agree

Reply:
Will be addressing this in a followup task.